docs/aquacomputer_d5next.rst

```diff
@@ -125,4 +126,5 @@ Debugfs entries
 serial_number    Serial number of the device
 firmware_version Version of installed firmware
 power_cycles     Count of how many times the device was powered on
+hw_version       Hardware version/revision of device
```

Comment:
Note that this is only supported on Aquaero

Reply:
Done, thanks!